pkg/config/config.go

```diff
@@ -1268,7 +1289,7 @@ func LoadConfig(configFile string) (cfg *Config, err error) {
 
 // NewConfig returns the default Config.
 func NewConfig() *Config {
-	return &Config{
+	c := &Config{
```

Comment:
out of curiosity, why this change?

Reply:
leftover from previous rollbacked refactor, restored